extensions/amp-script/0.1/amp-script.js

```diff
@@ -162,13 +165,14 @@ export class AmpScript extends AMP.BaseElement {
   /**
    * @override
    */
-  onMeasureChanged() {
-    if (this.layoutCompleted_) {
+  onLayoutMeasure() {
```

Comment:
Ideally `amp-script` would _always_ render, even if 0 size.
I'll spin out a quick PR now.

Reply:
on second thought, it likely makes sense for visual amp-scripts (ones not being used as a data-layer) to wait to be within load range before layoutCallback is called.